### DISCUSSION TITLE
Can I request my receipt of purchase?

### DISCUSSION THREAD

**Discussion by zephygrey** [aclap-dev/video-downloadhelper#312] [License & Purchase requests]
### What's your issue?

Sorry I'm not sure where to contact but I didn't get a receipt whenn I purchased premium. How can I get a copy of the invoice?
Cheers

### The 10 first digits (only 10) of your license number or email address

greyradian@hotmail.com

### I confirm that I did not not paste my full license number

- [x] I confirm

**Comment by jcfrog** [comment]
I sent you an email.

**Reply by mario-hernandez** [reply]
You can search for an email with subject "DownloadHelper video conversion license" sent by sales@downloadhelper.net


And at the bottom find: 
You can access your invoice at [https://www.downloadhelper.net/show-invoice?invoice=number]https://www.downloadhelper.net/show-invoice?invoice=number
